integration_tests/send_test.go

```diff
@@ -77,7 +75,7 @@ func TestXionSendPlatformFee(t *testing.T) {
 	//	t, client, network)
 
 	// Prep Interchain
-	const ibcPath = "xion-osmo-dungeon-test"
+	// const ibcPath = "xion-osmo-dungeon-test"
```

Comment:
If we are extracting into a specific function, we should probably just remove the commented out interchain parts